spec/amp-framework-hosting.md

```diff
@@ -0,0 +1,263 @@
+# Hosting the AMP framework
+
+You can host the AMP framework and components from your own server or CDN. This feature has a number of applications. For example, you can...
+
+- test and demonstrate changes to the framework or components.
```

Comment:
supernit: I would put this bullet point last

Reply:
Done